### PR TITLE
Lowers Armor Values

### PR DESCRIPTION
**Pull request by ghost**
Right now a basic security officer blocks 50% of all melee damage. It
would take 7 hits from an esword/punches from a Juggernaut to reliably
down an officer, and just a single stun baton to down a guy with an
esword. A wraith or any simple mob with lesser damage is basically helpless (which I'm obviously biased about, as I've added many of the simple mob varieties and plan to add more of them for migrants/Away Missions 2. The mission won't work if some guy in riot armor can just walk around immune to hostile mobs)

It takes 20 hits from an inactive baton to crit a basic officer, 8 hits
from an armblade, etc.

Basically melee is completely awful as soon as someone throws on a basic
vest. It makes melee weapons impossible to balance because even if
they're strong enough to crit a random med doctor in 4 hits they're limp
noodles vs a helmet. We can't move away from "I stun you lose" combat if
stun weapons are 8-20 times as effective as anything else. With this change they'll still be four times as strong, but it won't be as completely skewed.

I lowered the values around half for melee and tweaked laser resist a
bit as well (again, it shouldn't take 10 lasers to down someone vs their
one taser). I'm not overly attached to these values, nor are they
particularly deeply thought about (some are probably inconsistent, I did this in about 10 minutes),  I just wanted to get a conversation on git started where everyone would see it.

### CODE REVIEW

**Comment by Xhuis** [comment]
I don't see the reason to do this. You act like the armor isn't possible to remove, when you can easily just stun and quickly remove their helmet/armor. At the very least, if this does go through, I'd rather you kept the armor values on the mining hardsuit the same. As a frequent miner, the melee protection is necessary to not die very quickly from slip-ups.

**Comment by KorPhaeron** [comment]
@Xhuis "Easily stun and remove" misses the entire point of the pull request. Eswords are not stun weapons. Constructs are not stun weapons. A stick with a battery taped to it should not be a requirement for using a lightsaber.

I could see the issues with mining though, it's why I only lowered that a little bit. Probably wouldn't be the end of the world if it went back up though.

**Comment by Incoming5643** [comment]
friendly reminder that with goliath plates that miner hardsuits can reach some outrageous levels of brute resist, like 80 or 90.

I wonder if they'd start healing on hit if they passed 100

**Comment by Iamgoofball** [comment]
@Incoming5643 test it

**Comment by Fox-McCloud** [comment]
Armor values were rewritten to be non-RNG..that said, there was an extremely lengthy mathematical evaluation of armor the proved that the OVERALL damage blocked by armor was exactly the same.

I'll see if I can find it, but if the math bears it out, then this PR is literally worthless.

@Incoming5643 

It doesn't heal them; I've tried that before, in the past.

**Comment by KorPhaeron** [comment]
I stand corrected then. I still believe they are far too high based on the fact that eswords and the like are useless vs someone wearing a basic vest

**Comment by Fox-McCloud** [comment]
If they're wearing armor, then target their legs and arms, as they aren't covered; problem largely solved...

**Comment by Iamgoofball** [comment]
@Fox-McCloud limbs have a damage cap, and when you hit that cap I think it starts shoving damage off onto a nearby part, so if you cap out leg damage you might get your damage redirected to chest, which is usually armored

**Comment by KorPhaeron** [comment]
Not to mention that you don't always hit what you're aiming for.

And jackboots cover your legs for 50% damage reduction somehow. Most powerful boots in the universe.

**Comment by Fox-McCloud** [comment]
@Iamgoofball 

That's only on Bay; on TG, the damage just stops, AFAIK and you have to actually switch limbs, manually.

@KorPhaeron 

I don't see how that really impacts anything; you hit your desired target 80% of the time...

Jackboots have no armor values.

**Comment by Lumipharon** [comment]
Other then one time where someone complained about captain's armor for literally an hour, I don't think I ever see people complain about armor being too strong.

You use the e-sword example - if you're fighting a sec officer, then they have stun weapons on them, so no matter what, the stun has the technical advantage - the whole point of the armor is that people CAN'T just bruteforce the shit out of them.

Juggs only need to get 2-3 hits in, to slow the person down then they're fucked. Where as the jugg itself is reflecting lasers, stun immune, and immune to the vast majority of melee attacks. Hm?

**Comment by sirbayer** [comment]
It appears I actually disposed of those files; I apologize. I can try to redo the math if you folks like.

**Comment by Fox-McCloud** [comment]
Up to you; I don't personally have need of it.

Side note, thank for making this system; we implemented it on our own codebase, and it's light-years better than the old "lolrng" style armor.

**Comment by KorPhaeron** [comment]
You don't need to do the math, I believe you. Maybe I just notice melee weapons being weaker now that less people have access to stuns. Like people keep saying, armor is useless against tasers/batons, so I didn't notice back when everyone had stungloves and taser spraying eguns.

**Comment by sirbayer** [comment]
Alright, I'll leave you folks to it.

**Comment by WJohn** [comment]
This is an absolutely gigantic nerf and I really don't see the need for it. Armor is not all that common and it should be actually able to do its job when it's restricted to body parts.

**Comment by freerealestate** [comment]
If only armour values scaled with security population...

**Comment by xxalpha** [comment]
I agree with this nerf.

**Comment by KorPhaeron** [comment]
@WJohn 

I don't believe the nerf is gigantic. As others have pointed out, using a stun weapon renders the armor worthless as you can strip it right after your attack completely bypasses said armor. This just makes non stun weapons slightly more viable.

**Comment by WJohn** [comment]
Uh, deathsquad armor and the admiral trenchcoat and such should not be nerfed any, they're meant to be nearly unstoppable to all forces including melee.

**Comment by Cookiesndip** [comment]
I have mixed feelings on this.

On one end, non-stun melee weapons getting a buff against sec is nice. On the other hand, nerfing specialty equipment (riot suits, bulletproof vests, etc.) against their damage type seems counterproductive. Would you ever wear a riot suit in cult any more if it only gave you 10~ more melee resist, instead of 20~? Keep those those the same, imho.

Also, general armor is restricted to certain bodyparts, so they're balanced in that sense. Rather, I think the solution is to remove the limb damage cap. the only reason it existed in the past was to make it so you couldn't get instacrit by doors, but it now distributes across your entire body so I see no reason not to. Thus, you can just whack at sec's legs for reduced, but decent damage.

Furthermore, I do not think it's a good idea to nerf laser armor. Yes, tasers have some advantage if you're near someone, but they're limited range, not spammable, and don't go through glass. Lasers do, have much more ammunition, deals semi-permanent damage, AND slows, making it easier to hit the more you hit someone. Disablers are a tertiary option that people only use if they're shooting through glass and want to keep folks alive, are spamming down maintenance with 'em, or they can't aim, given that it's all stam damage and the stun lasts fucking nothing, they're a purely lesser option to lasers.

Regardless, there's the manner of nuke ops. They _need_ that armor to deal with general crew. Throw them a goddamn bone here.

Imho, half generic sec armor to melee only, keep the HoS/Captain's equipment armor values the same (they ARE the most important roles on the station after all - they better damn well have robust armor), keep nuke op armor values the same and call it a day. Anything more is excessive, to be honest.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	desc = "High speed, low drag combat boots."
 	icon_state = "jackboots"
 	item_state = "jackboots"
-	armor = list(melee = 50, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 10, rad = 0)
+	armor = list(melee = 25, bullet = 25, laser = 25, energy = 25, bomb = 50, bio = 10, rad = 0)
```
Comment (Anonus) [inline code/modules/clothing/shoes/miscellaneous.dm]: I'm pretty sure these values never apply to anything at all!

**Comment by Anonus** [comment]
Antag armor should still be pretty strong, as it's just a few antags against a station of validhunters and security.

**Comment by SconesC** [comment]
Armor values exist so that you are not able to brute force Security officers. You make it sound like it should be a valid tactic to run at an Officer and try to esword him to death or something.

Also yeah those combat boot values apply to nothing

@Anonus What armor, even? Cult robes?

**Comment by Lumipharon** [comment]
Who ever complains about armor anyway?

This is really just a nerf to sec more then anything else - stuns have already been nerfed to shit, they have a range of 7, which mean if you're chasin a guy using stuns, they can literally shoot you while it's impossible for you to hit them back.

Not to mention you can't hit people off screen, where as all non stuns can.

If you're running up to people in armor and are suprised you're not able to ggnore them with an e-sword or armblade, then I unno what to say.

**Comment by KorPhaeron** [comment]
I'd rather security get real tasers back honestly.

And yes I believe rushing someone with an energy sword should be dangerous for that other person. If they're running around with an esword out the downside is that they're attracting a lot of attention. That or you ran into them in maint, in which case being ambushed in maint by a guy with a sword should be a big deal.

I think stuff other than emp flashlight+stunprod should be capable of killing people. 

I don't understand how people are simultaneously complaining that I'm making armor too weak and saying that armor already sucks because it doesn't stop stuns. If we're gonna let one hit "kills" (stunprod, stun baton, taser, flashbang) stay in the game what is wrong with non stun weapons being viable as well?

Seriously, a battery on a stick is a bigger threat to people than a swarm of space carp or a spooky wraith

**Comment by KorPhaeron** [comment]
@SconesC 

It should be a bad tactic because the officer has better weapons and will likely yell for backup, not because you can't hurt him effectively. Individual officers should not be like town guards in an MMO that auto win to prevent all crime, or the rounds become incredibly slow and stagnant when traitors have to either stay extremely quiet or get robusted immediately

**Comment by KorPhaeron** [comment]
@WJohn 

Admins can give them god mode if they really want them to take no damage. I still maintain it isn't fun to have invincible people running around clicking everyone to death. You're taking away control of the game from a majority of players. Just send another deathsquad if the first one dies somehow

**Comment by Lumipharon** [comment]
Because you just want non stun weapons to be effective as stun weapons for ending a fight.

That is what stun weapons are for, and all antags can get their hands on them.

Damaging weapons are for KILLING people, which is why officers don't have lethal guns.

**Comment by xxalpha** [comment]
What stun weapons can antags buy?

**Comment by freerealestate** [comment]
uhhhhh

the uh

the one that fires riot foam darts

yeah. that one

**Comment by Man-Shroom** [comment]
The ebow.

Personally, I agree with this nerf. A lot of the game has been nerfed over the years in various ways, an yet armour has stayed the same, or even gotten stronger since the removal of RNG armor. For balance, we should either nerf armor, or buff everything else.

**Comment by Cheridan** [comment]
To naysayers: Would it be fair if we reduced the values, but increased the amount of time it takes to strip armor? To fix the whole "stuns > armor because you can just strip it off" thing.

**Comment by Anonus** [comment]
My personal opinion here: one-click stuns are maximum bullshit and honestly not really fun to deal with.

**Comment by KorPhaeron** [comment]
@Cheridan 

Most stuns can just re-apply the stun forever so I'm not sure how much it would help, but I like the compromise in general

**Comment by WJohn** [comment]
Honestly if we are to move away from insta stuns and instead move towards stamina damage (which is affected by armor) then I think this would be a poor change for that at that point, since the game WOULD revolve more around armor than stunning.

Honestly I don't think we should be giving more credit and more incentive to insta-stuns if we all want it gone by making armor just subdued as if to say "stuns are the only combat mechanic and this isn't going to change".

**Comment by KorPhaeron** [comment]
@WJohn 

I'd rather work with what we have now than worry about what we're implying in regards to some far off, hypothetical future

**Comment by Lumipharon** [comment]
Revolvers are literally still 2 hit crits on sec officers with armor you know.

With stuns, if you miss the first shot (since it has range and recast limits), you're at a much bigger disadvantage against a guy with a normal gun, which you can spam.

Armor strip time already got buffed tremendously, and stuns have already been nerfed repeatedly, rather hard.

I don't understand how armor could not be an issue then, yet become an issue now, when non stuns are more viable then ever.

**Comment by xxalpha** [comment]
Revolvers aren't that good. Go ahead and test it.

**Comment by KorPhaeron** [comment]
Revolvers aren't really comparable at all to melee weapons. Not everyone has a revolver either.

**Comment by Fox-McCloud** [comment]
Revolvers are good. 

Officers have 15 bullet resist on their heads and chest....enough to reduce a bullet's damage from 60 to 51---an officer is still going to get two-shot ....like everyone else, unless he's wearing bullet proof all the time.

**Comment by Lumipharon** [comment]
How are they not good XXalpha?

**Comment by freerealestate** [comment]
I'm kind of torn about these. I think security definitely needs to be
weaker than it currently is in some situations, but also stronger in
others. When there are few security officers, each needs to be stronger to
cope. As I implied earlier, I think security would be more balanced if
officers spawned with better armour the higher the ratio of security to
crew was. Presumably difficult to code, and probably would never happen,
but I think population is still relevant to talk about.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -27,7 +27,7 @@
 	desc = "High speed, no drag combat boots."
 	permeability_coefficient = 0.01
 	flags = NOSLIP
-	armor = list(melee = 80, bullet = 60, laser = 50, energy = 50, bomb = 50, bio = 30, rad = 30)
+	armor = list(melee = 40, bullet = 30, laser = 25, energy = 25, bomb = 50, bio = 30, rad = 30)
```
Comment (Iamgoofball) [inline code/modules/clothing/shoes/miscellaneous.dm]: These are deathsquad boots, revert

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 3]
```diff
@@ -68,7 +68,7 @@
 	flags_inv = 0
 	w_class = 3
 	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50,energy = 50, bomb = 100, bio = 100, rad = 100)
+	armor = list(melee = 50, bullet = 40, laser = 30,energy = 50, bomb = 50, bio = 100, rad = 100)
```
Comment (Iamgoofball) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Deathsquad helm, revert

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 4]
```diff
@@ -304,7 +304,7 @@
 	flags = NODROP
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	slowdown = 2
-	armor = list(melee = 65, bullet = 20, laser = 10, energy = 13, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 40, bullet = 20, laser = 30, energy = 13, bomb = 0, bio = 0, rad = 0)
```
Comment (Iamgoofball) [inline code/game/gamemodes/changeling/powers/mutations.dm]: This should be buffed, really, considering it gives you slowdown 2

Comment (KorPhaeron) [inline code/game/gamemodes/changeling/powers/mutations.dm]: The slowdown should probably just be removed, didn't notice how awful it was

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 3 above)
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: I guess deathsquads and whether they should be mortal or not is a completely different argument from normal armor values. I still think invincible mobs shouldn't be a regular thing.

Comment (Dorsisdwarf) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: >80 melee armor
>invincible

File: code/game/gamemodes/changeling/powers/mutations.dm (hunk 4 above)
Comment (SconesC) [inline code/game/gamemodes/changeling/powers/mutations.dm]: put it to slowdown 1

**Comment by optimumtact** [comment]
probably okay, we can always buff things later if we fucked up :+1: 

also :+1: to leaving deathsquad related values alone, no need to nerf adminspawned tanks

**Comment by Lumipharon** [comment]
Look at this at the logical extreme.

You're saying that armor makes non stunning weapons shit compared to stun weapons, since stuns are typically '1 hit, game over'. non stun weapons are 'multiple hit to crit'.

But lets take it to the logical end point.

Lets imagine nothing gives you armor - nothing at all.
Non stun weapons still require more then a single hit to down someone, and stun weapons are still are a 1 hit stun.

So it doesn't change the issue that you're ultimately trying to actually address.
All it does, is mean certain weapons (basically only melee) are more effective on security, but in melee officers already have batons. In range, sec's bullet resist is so poor that only the HoS can take 2 hits from a revolver, and even then only barely.

If you have an issue with stuns even after how much they've been nerfed in the last year or so, address that.
Armor only helps with general survivibility - reducing the effectiveness of armor in no way reduces the effective 'takedown' speed of stuns vs non stun weapons.

**Comment by xxalpha** [comment]
Why should a common stun baton be better at melee than a melee weapon that costs more than half of a traitor's TC?

**Comment by Anonus** [comment]
The flipside on that is, well, how do you make the stunbaton worse without just making it useless?

**Comment by xxalpha** [comment]
You don't.

**Comment by Jalleo** [comment]
We cant achieve a perfect balanve we need something to be imbalanced we need some proper data.
This whole thing is credible but with how things are we are basically at a point where its down to trying to maintain a almost perfect balance on quicksand.
This discussion is really how much should we be trying to control the fundemental basis of this game with options that are becoming redundant but more required than ever.

**Comment by Lumipharon** [comment]
@xxalpha Why should a common taser be better then a revolver that costs more then half of a traitors tc's?

They're not comparable, is why.
Stun weapons are for unharmfully take downs. Bein able to bludgeon people to death with them is concidental.

Revolvers/e-sword/other non stun weapons are for killing people quickly, not stunning.
If you get someone with a taser/e-bow, sure, the guy might be fucked in terms of his elite 1v1 pvp fight, but he still has a chance to scream for help as he gets chain stunned, cuffed, stripped and killed.

Whipping out a revolver and doubling tapping someone crits them before they can so much as react.

**Comment by xxalpha** [comment]
But you will loose if you try to fight a sec officer who is wielding a stunbaton. As soon as you get close you go down.

Unless the eswords block them, then disregard me.

**Comment by Lumipharon** [comment]
And my whole point is, regardless of armor, the stun baton still disables a person faster then an e-sword.
An e-sword is like 3-4 hits to crit with no armor (I think the single handed is 4, dual is 34 damage so 3).
Except random KO's, but fuck those.

So yes, if you slashed armor values this gap would be marginally smaller, but still very much there.

**Comment by xxalpha** [comment]
Honestly eswords should probably go through all armor and also act as shields, then we wouldn't have this armor problem as there would be a counter to it.

**Comment by Lumipharon** [comment]
They already have a reasonable block chance, but having armor penetration would be neat - and would make sense for what e-swords are supposed to be.

Would it be possible, as an opposite of armor, to have armor penetration values for weapons?
So X weapon would have a AP of (-)20%, either has a multiplier or additive, to the targets armor.

**Comment by xxalpha** [comment]
Eh not something as complicated. Just a few select weapons ignoring armor is enough. The only two I can think of that  should have this ability are the esword and pulse guns.

Also I meant eswords shielding you from batons. 100% chance or close to it.

**Comment by Lumipharon** [comment]
Then dual sword with no slips is nigh unbeatable.

**Comment by xxalpha** [comment]
Can you still buy dual esword plus noslips? And I think that the esword should rule in melee range, being beaten by a stunbaton, which every sec officer spawns with, doesn't feel right. At the very least it should have a chance to block melee attacks.

**Comment by Lumipharon** [comment]
Yes you can, it takes up 20 tc's.

E-sword already has a block chance, but if you buffed it a lot it just becomes an uncounterable piece of shit that lets you just run down the halls murderboning as fast as possible.

If the sword had full armor ignore, you could quite possibly push a sec officer down and murder them before they could get up, pull out a baton, turn it on and hit you with it. Adding a ggnore block to that is not great.

The thing with adding a percentage/flat armor reduction, (if it can get coded, I unno), is that it's more flexible. You could make weapons 100% armor piercing, not at all, or somewhere in between. Makes it easier to balance things.

**Comment by xxalpha** [comment]
I disagree with percentages, it doesn't need to be complicated. A simple armor ignore is easier to balance.

**Comment by SconesC** [comment]
How would you kill mythical double esword guy

Traitor is a round about espionage, that's why we don't have a fuckton of assault weapons and body armor in the uplink

**Comment by Steelpoint** [comment]
Running up to someone with a Esword is a valid tactic as a Traitor against anyone outside of Security.

Security are meant to be the primary counter to antagonists. Nerfing their armour simply means your making it far easier (than it already is) to just run to a Officer and slash them up quickly.

**Comment by Steelpoint** [comment]
Furthermore some of these changes outright make some armour useless.

Riot and Bulletproof Armour are suddenly wholly ineffective in their tasks, what's the point of wearing a armour dedicated to defending against damage type X if said defence is mediocre at best and yet you still suffer the disadvantages of wearing said armour, meaning you have even worse defence on all other damage types as well as suit slowdown effects.

Deathsquad Armour and ERT Armour do NOT need this nerf in any way, Deathsquad even more so as the whole concept behind them is that they are meant to be extremely hard to kill.

The Ling armour is already woefully inadequate and this change makes the armour utterly more worthless than it already is.

Amusingly you're buffing the Captains armour? Is this intentional?

**Comment by KorPhaeron** [comment]
I don't think an effective doubling of their health vs common weapons is an "edge"

I don't think armor being weaker would make eswords uncounterable either, you can always baton, flashbang, aoe flash, pepperspray, or slip someone like that.

Armor ignore might be interesting but it doesn't really fix the matter of common weapons just bouncing off of people with armor. 

I'm not saying fire extinguishers should be the robust weapon of choice but people smashing you over the head with one should at least be concerning. Revs/gangs/cult/simple animals all often rely on brute melee, buffing random bits of traitor gear won't solve that problem.

**Comment by optimumtact** [comment]
Are people amenable to discussing this in the forum? github's font size and css layout doesn't really lend itself to in depth discussion of the pros and cons of the change, Its really hard for me to keep track of what is what

**Comment by xxalpha** [comment]
>Security are meant to be the primary counter to antagonists. Nerfing their armour simply means your making it far easier (than it already is) to just run to a Officer and slash them up quickly.

That's fine and all, but eswords cost way more than a stunbaton, therefore they should rule.

**Comment by Steelpoint** [comment]
A piece of soap costs more than a stunbaton, cost is irrelevant considering security get their equipment as standard issue.

**Comment by Dorsisdwarf** [comment]
Soap is a deadlier weapon than an esword, because it stuns.

Also "but I-i-i wont be able to 1-hit-noscope the dual esworder if he gets baton resistance" is a shitty idea. Not only are they spending 90% of their TC on the weapon you want one click from standard-issue kit to counter, but they're at risk of suicidisstants disarming them (because again, it only takes one shove, but three hits on an unarmored suspect.
If you really can't think of how to deal with them, here's a hint. /all the projectile weapons security starts with, basically specifically for this eventuality/

**Comment by SconesC** [comment]
>use the energy weapons against dual swords?

am i misreading that? its pretty late, i might be honestly

also if they could block melees what would you propose to kill them? if they killed a sec officer there would be 3 guns on the station capable of killing them and the chance of a random officer having those (the riot guns and the det revolver) is nill.

trying to disarm a dual esword should just damage the guy trying to do it, they really should be immune to that crap

**Comment by xxalpha** [comment]
Grenades.

**Comment by Lumipharon** [comment]
first thing any murderboner can and will do it hit the armory. Kill the warden, get flashbang protection, space the shotguns and the station is fucked.

**Comment by KorPhaeron** [comment]
Where did this "eswords should make you immune to all attacks" discussion come from anyway I just wanted them to hit harder

**Comment by Steelpoint** [comment]
They already do hit very hard against unarmoured opponents, which equates to the majority of the station.

**Comment by KorPhaeron** [comment]
@steelpoint

Exactly which is why making the weapon itself stronger would be bad, thus the PR

**Comment by Razharas** [comment]
I agree changeling armor doesnt need slowdown if tis nerfed
Remove it and ill merge

**Comment by Fox-McCloud** [comment]
Wait..this is actually going through? It's been statistically proven that uh armor is the same as it was before---especially given the negativity towards making "Murderboning" easier and such?

**Comment by KorPhaeron** [comment]
@razharas 

Would you accept making the slowdown 1 (or removing it entirely) instead? Otherwise I'll do what you said

**Comment by Lumipharon** [comment]
Can we jsut not alter armor? if the issue is with stuns then it's an issue with stuns - nerfing armor doesn't fix that, as we've already looked at.

**Comment by KorPhaeron** [comment]
@Lumipharon

If we magically removed stuns tommorow I still think armor granting what is essentially 200% health for no drawback would be a bad idea

**Comment by Lumipharon** [comment]
against a specific damage type, with an item that covers only some parts of the body, which is in limited supply.

**Comment by Iamgoofball** [comment]
@KorPhaeron remove the slowdown if the armor gets nerfed dude

**Comment by WJohn** [comment]
I'm still not for nerfing armor when this'll ultimately have to be undone for that one day when someone finally changes combat to be less about insta-stuns and more about a no-holds-barred beatdown.

**Comment by optimumtact** [comment]
@WJohn I don't think trying to take into account things that may or may not happen in the distant future is a good strategy.

**Comment by Razharas** [comment]
@KorPhaeron 
Preferably remove, because armour gives you away 100% while slowing you down considerable yet not giving you much combat benefits

**Comment by KorPhaeron** [comment]
Alright @Razharas it's ready, but the values will likely need to be revisted/tweaked in further PRs

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 5]
```diff
@@ -40,7 +40,7 @@
 	icon_state = "deathsquad"
 	item_state = "swat_suit"
 	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50,energy = 50, bomb = 100, bio = 100, rad = 100)
+	armor = list(melee = 50, bullet = 40, laser = 30,energy = 50, bomb = 50, bio = 100, rad = 100)
```
Comment (GunHog) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: To be honest, this absolutely should not be merged until the Deathsquad changes are removed. This is an admin event, for when ERT cannot cut it. They are expected to deal with several threats at once. Inferior armor will break this.

Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: @GunHog 

They still basically kill anyone in 2 shots and admins can spawn infinite of them if the first wave dies somehow.

Comment (freerealestate) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: If death squads (and aliens) were more balanced we'd be able to play them
more often.

**Comment by Steelpoint** [comment]
I still disagree with this change over all, all this does is make it take a few less hits to kill a stunned security officer as well as remove any chance of said officer recovering from the stun and being able to escape.

The few scenarios where you see a nearly dead officer managing to escape from someone will never occur again.

if you want to tackle any problem with SS13's combat mechanics you need to look deeper beyond the armour values and look at how actual combat occurs, and if you do look you'll see the problem is not with stuns but with the fact that any other form of combat outside of stuns is useless, unless you have weapons that knock someone into crit in one hit then nothing will trump stuns.

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 5 above)
Comment (Miauw62) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: >If death squads (and aliens) were more balanced we'd be able to play them more often.

Comment (freerealestate) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Nice meme! You can't deny the truth though.

**Comment by Razharas** [comment]
Welp looks ok to me

File: code/modules/clothing/suits/armor.dm [hunk 6]
```diff
@@ -57,7 +57,7 @@
 	icon_state = "capcarapace"
 	item_state = "armor"
 	body_parts_covered = CHEST|GROIN
-	armor = list(melee = 50, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
+	armor = list(melee = 50, bullet = 40, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
```
Comment (bear1ake) [inline code/modules/clothing/suits/armor.dm]: WTF?????

**Comment by Steelpoint** [comment]
I still think this PR is nothing more than a meaningless change that changes little. Aside from guaranteeing that anyone who gets stunned will die before they can get back up of course.

**Comment by KorPhaeron** [comment]
@Steelpoint 

That's more or less true already since you can infinitely re-apply stuns

**Comment by SconesC** [comment]
These HoS nerfs make me so uncomfortable

I like what's being done here conceptually but combat is stun-based and this just = less time on the ground to scream for help/no chance of getting back up

I guess I'm gonna stop playing Security, because this is going to put them at borderline unplayability. I can perhaps understand your frustration with melee but roles like HoS are now going to be dead in the water with their traditionally extremely powerful armor stripped down to be only barely above that of the rank and file officer - Nuke ops can tear Security a new one without bulletproofs currently (Or pre-merge, I guess). I know this is a poor time to write a dissertation on the matter because we pretty much never revert things, but I'd just like to say everything now so I don't have to complain in OOC/feedback for months. 

This is an extremely poor time to apply this change. Summer is real. >50% of Security players I see are still struggling with basic controls and mechanics. Older players with antag enabled are already ripping through them no problem - And again, this really just means less time on the ground to scream for help. Combat revolves entirely around stuns because of our medical system, and armor exists so that you cannot brute force a security officer aka winning a fight without a stun.

I don't know, man. I guess we'll see how it goes.

**Comment by SconesC** [comment]
Also energy resist is a sick joke, no antags have energy weapons unless they steal from the Armory

**Comment by KorPhaeron** [comment]
@SconesC 

I can always hit the revert button or tweak the values upward, it's very little work to do so.

Maybe we should just go ahead and change the rest of the stuns to stamina of some sort instead of this godawful halfway point we have now, that might have been a better fix than this armor thing

**Comment by SconesC** [comment]
HoS should be outshining officers by a significant margin - It's the hardest role in the game and really benefits from having equipment to make the job possible.

I'd like to discuss stun alternatives that are not LIMBS EVERYWHERE FUCK STUNS YOU STAND TIL YOU BLEED OUT even though that's what I want, but I think here is not the place.

**Comment by KorPhaeron** [comment]
>LIMBS EVERYWHERE FUCK STUNS YOU STAND TIL YOU BLEED OUT

That'd be amazing though

**Comment by SconesC** [comment]
Yes, it's what I want, but I know I have to be realistic in progressively fixing this issue.

Stun-based combat is trash. I do not mind a game of absolutes - I am very good at it. I just find RNG hyperlethal to be more interesting.

I think we have a thread for this.

But yeah HoS stuff needs upwards tuning, it's fucking important to have at least someone who is a terrifying blackcoat nazi tank with three guns and a lust for valids. If he is weak, Security is weak.

**Comment by Steelpoint** [comment]
Yeah, the current HoS armour values are a joke. Its not even a step up above normal armour.

**Comment by SconesC** [comment]
IF THE ALPHA IS WEAK THE PACK WILL FAIL

**Comment by GunHog** [comment]
This really should not have been merged.

**Comment by Saltycut** [comment]
Oh God, this is such a massive nerf. I don't like it at all, it wasn't needed.

**Comment by SconesC** [comment]
We'll see how it goes. While I personally detest it I'll play some rounds and see how drastically if effects me getting fucked, and talk with Kor to make proper adjustments.

**Comment by Lumipharon** [comment]
Whelp, time to not bother with sec.

**Comment by optimumtact** [comment]
Look at all these mad security players

**Comment by Lumipharon** [comment]
Notice how security are 95% of the people that have armor? People that routinely (regardless of roundtype or antagonist) get the shit beaten out of them by non stun weapons and only survive because armor?

**Comment by GunHog** [comment]
Mad security players? Maybe because they are the only ones with armor. An armor nerf is a Sec nerf, and given how harsh it is to play Sec these days, I do not see this change going well at all for those poor players.

**Comment by CPTANT** [comment]
This is a completely unnecessary nerf to security. The only times security really benefits from their melee resist is during rev/cult when they are usually vastly outnumbered anyway. 

The dominance of stun combat should be adressed by slightly lowering stun times and moving towards stamina based stuns as much as possible.

**Comment by optimumtact** [comment]
I just can't handle all this mad

**Comment by Steelpoint** [comment]
People disagree with a change and you call them salty?

Very professional.

**Comment by CPTANT** [comment]
You could just as well call the "boohoo armor too stronk" camp whiners in the first place. But since that is non-constructive as fuck I have no idea why anyone should bother.

The ONE example named to justify this nerf is e-swords being too weak to take on sec. You could have done anything from removing limb damage caps to buffing traitors instead of mutilating sec gear.

**Pull request merged**